.drone.yml

```
@@ -27,7 +27,7 @@ steps:
 
   - name: lint-backend
     pull: always
-    image: golang:1.16
+    image: gitea/test_env:linux-arm64
```

It'd be nice to have a link to the repo so people know where to look for modifications.
```suggestion
    # https://gitea.com/gitea/test-env
    image: gitea/test_env:linux-arm64
```